src/components/biblio/topic_entity_tag/TopicEntityTable.js

```diff
@@ -32,9 +35,10 @@ const TopicEntityTable = () => {
   const [selectedSpecies, setSelectedSpecies] = useState([]);
   const [speciesFilterPosition, setSpeciesFilterPosition] = useState({ top: 0, left: 0 });
   const [allSpecies, setAllSpecies] = useState([]);
-  // set the initial pageSize state to 25
-  const [pageSize, setPageSize] = useState(25);
-    
+  const defaultPageSize = 25;
```

Comment:
I don't think you need defaultPageSize in the component, because you're initializing it the reducer, and getting it from storedPageSize, so you could just use that instead of these 2 + pageSize

Reply:
@azurebrd Fixed. =)